scripts/migrations/007-experimental-false.ts

```diff
@@ -26,7 +26,7 @@ const dirname = fileURLToPath(new URL('.', import.meta.url));
  */
 export const fixExperimental = (bcd: CompatData | Identifier): void => {
   for (const { compat } of walk(undefined, bcd)) {
-    if (!compat?.status?.experimental) {
+    if (!compat.status?.experimental) {
```

Comment:
See https://github.com/mdn/browser-compat-data/issues/17258 about the remaining question mark here.